frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -647,7 +647,16 @@ export function doInitPings(
     // not to do so as it's semantically cleaner to not give the healthcheck
     // endpoint additional responsibilities.
     Promise.all([
-      axios.get(healthzUri, { timeout: minimumTimeoutMs }),
+      // NOTE: We temporarily avoid hitting the healthz endpoint for now
+      // because certain environments (notably GCP App Engine and Cloud Run)
+      // reserve the endpoint name, and the new /st-allowed-message-origins
+      // can be used as a healthcheck at the relatively cheap cost of some
+      // semantic clarity.
+      //
+      // Once we're able to pick up work on https://github.com/streamlit/streamlit/pull/5534
+      // again, our endpoints can be re-split into their original dedicated
+      // roles.
+      Promise.resolve(), // axios.get(healthzUri, { timeout: minimumTimeoutMs }),
       axios.get(allowedOriginsUri, { timeout: minimumTimeoutMs }),
```

Comment:
If this were intended to be long-lived code, we'd really want a `WebsocketConnection` test as well ("test that we can connect to the backend even if `healthz` does not respond"). How tricky is that to add?  Basically: is it a small enough amount of work that we should include the test in this PR to prevent a regression sneaking in before the healthz endpoint issue is resolved?

Reply:
I wasn't able to figure out a reasonable way to write a test ensuring that we can still connect to the backend if `/healthz` doesn't respond, but I think in this case it'll be sufficient to write a test asserting that we don't ping `/healthz` from `doInitPings` at all.

I also added a comment mentioning why we're keeping the `Promise.all` structure even though the first element of the array passed into it is just an already-resolved promise.